tests/internal/anvil/framework.go

```diff
@@ -373,7 +373,8 @@ type ThingGatewayTestState struct {
 }
 
 func (i *ThingGatewayTestState) SetGatewayTree(tree string) {
-	i.ThingGateway.AuthenticateWith(tree)
+	// TODO fix tests using exposed internal method
```

Comment:
Setting the auth tree on the gateway is no longer possible. I'll fix the tests if we're happy with the rest of the changes in the PR. All the AM SDK tests pass.

Reply:
👍